parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -63,6 +63,9 @@ if [[ "${DISABLE_SSH}" == "true" ]]; then
     disableSSH || exit $ERR_DISABLE_SSH
 fi
 
+apt update && apt install -yq tinyproxy
```

Comment:
this works but doesn't handle https serving endpoint for HTTPS_PROXY

you can set HTTPS_PROXY to http://url, but it's not testing full scenario